src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -329,8 +330,20 @@ int fx_muxer_t::execute(const int argc, const pal::char_t* argv[])
             else
             {
                 trace::verbose(_X("Executing as a standalone app as per config file [%s]"), config_file.c_str());
-                corehost_init_t init(deps_file, probe_path, _X(""), host_mode_t::muxer, &config);
                 pal::string_t impl_dir = get_directory(app_or_deps);
+                if (!library_exists_in_dir(impl_dir, LIBHOSTPOLICY_NAME, nullptr) && !probe_path.empty() && !deps_file.empty())
```

Comment:
This does not sound right. Hostpolicy.dll should be loaded from the FX directory corresponding to the FX called out as app dependency. Why do we need this change?